recipes/watchdog-wms/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "watchdog-wms" %}
+{% set version = "1.2.3b" %}
+
+about:
+    home: https://www.bio.ifi.lmu.de/watchdog https://github.com/klugem/watchdog
+    license: GNU General Public License, Version 3
+    summary: 
+       Watchdog, a WMS for the automated and distributed analysis of large-scale experimental data. 
+       The software is implemented in Java and is thus platform-independent. 
+       Main feature include
+        - straightforward processing of replicate data
+        - support for distributed computer systems
+        - customizable error detection
+        - manual intervention into workflow execution
+        - GUI for workflow construction using pre-defined modules
+        - a helper script for creating new module definitions
+        - no restriction to specific programming languages
+        - provides a flexible plugin system for extending without modifying the original sources
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+build:
+  number: 0
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/klugem/watchdog/archive/v{{ version }}.tar.gz
+    sha256: 7ec16713fcf58dad0bb474e2fe7ded3064d86e48e9baf2408ec9a69be5e580db
+
+requirements:
+  build:
+      - openjdk >=8
+      - coreutils >=8
+  run:
+      - openjdk >=8
+      - coreutils >=8
+
+test:
+  files:
```

Comment:
Please add a test case. This can be trivial, e.g. calling --help.

Reply:
@bgruening are test files supported by mulled? Previously, they weren't if I remember correctly.

Reply:
Yes, currently not supported, sorry.

Reply:
Thanks @bgruening. @bneron, looks like these files haven't been added by you, right? They are in fact already in the master branch. Seems like github has a hiccup here. I think this is ready to merge.